examples/using-babel-plugin-macros/pages/posts-custom/index.js

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link'
+
+// the import-all.macro that we're using in /posts is cool, but the drawback is
+// that we're literally including all of the data/code for all the posts in that
+// file which may not be what we want.
+//
+// So let's write our own macro that's specific for our use case which will
+// replace all instances of `allPosts` with only data we need for this page
+// (path, and title).
+import allPosts from '../lib/import-posts.macro'
```

Comment:
The relative path is not correct here , it should be `../../lib/import-posts.macro`